digdag-standards/src/main/java/io/digdag/standards/command/ecs/EcsClientConfig.java

```diff
@@ -72,9 +73,17 @@ public static EcsClientConfig createFromTaskConfig(final Optional<String> cluste
             name = clusterName.get();
         }
 
+        String systemConfigPrefix;
+        if (ecsConfig.has("system_config_prefix")) {
+            systemConfigPrefix = ecsConfig.get("system_config_prefix", String.class);
```

Comment:
I am not sure the taskConfig generation flow but who (which system) can set "system_config_prefix" in ecsConfig and where is it described ? (server.properties?)
My concern is if users (not server administrator) can set this in workflow definition, it will cause of issues.